src/app/qgisapp.cpp

```diff
@@ -7690,7 +7691,58 @@ void QgisApp::legendLayerStretchUsingCurrentExtent()
   }
 }
 
+void QgisApp::applyStyleGroup()
```

Comment:
applyStyleGroup sounds strange to me (but I'm not a native english speaker). applyStyleToGroup() ?